beginner_source/quickstart/build_model_tutorial.py

```diff
@@ -0,0 +1,153 @@
+"""
+Build Model Tutorial
+=======================================
+"""
+
+###############################################
+# The data has been loaded and transformed we can now build the model. 
+# We will leverage `torch.nn <https://pytorch.org/docs/stable/nn.html>`_ 
+# predefined layers that Pytorch has that can both simplify our code, and  make it faster.
+# 
+# In the below example, for our FashionMNIT image dataset, we are using a `Sequential` 
+# container from class `torch.nn. Sequential <https://pytorch.org/docs/stable/generated/torch.nn.Sequential.html>`_ 
+# that allows us to define the model layers inline. 
```

Comment:
While this is certainly a quick way to chain modules in a container, I am not sure if we want to set this up as one of the "standard" ways of defining a model.

@ranman your take?

Reply:
I called the approach "inline" primarily because it is directly declared in code. I admit this is my own naming and happy to defer to your judgment. I also thought it important to show both methodologies as they are pretty common: it shows flexibility and helps folks see an explanation to what is already out in the wild.

Reply:
I am working through these feedback updates. Was there a decision here on what changes you thought should be made? 
1. Should I leave the "inline" term or do we want to remove/change it? 
2.  When you say "standard" way of building a model do you mean that for the main section of the code you prefer the object oriented class approach in the main tutorial vs "inline"?

Reply:
I'm not in favor of using the "inline" model in what is a tutorial for someone new to PyTorch. `nn.Sequential` is a container that is different in essence and functionality from a fully-equipped model. I am okay with it being used somewhere in more advanced tutorials, but not in a 101.

Reply:
Ok sounds good. I will make this change.